docs/announcement.md

```diff
@@ -0,0 +1,120 @@
+We annouce both community-ee-base and community-ee-minimal together
+
+# Forum Announcement
+
+Release announcement to be done  in the  [Ecosystem Releases category](https://forum.ansible.com/c/news/releases)  under the News and Announcement section.
+
+# E-Mail Announcement
+
+`ansible-annouce` Google Groups
+
+# Community-ee (both Base and Minimal) announcement
+
+```
+Hello everyone,
+
+We’re happy to announce the release of the
+
+Ansible Community Execution Environment Minimal <ansible-core-version-minimal-ee-version> and Ansible Community Execution Environment Base <ansible-core-version-base-ee-version>!
+
+
+## Whats inside community-ee-minimal <ansible-core-version-minimal-ee-version>?
+
+Ansible community-ee-minimal is a container image. It includes:
+
+- `base_image`: fedora-minimal
+
+- `ansible-core`: <version of ansible-core>
+
+- `collections`: The following set of collections
+
+    - ansible.posix <version>
+    - ansible.utils <version>
+    - ansible.windows <version>
+
+## sha256 sum value of the container image:
+
+
+
+## How to get community-ee-minimal?
+
+### Install from command line via @sha256_sum_value of the image
+
+`podman pull http://ghcr.io/ansible-community/community-ee-minimal@sha256:sha_value`
+
+### Install from command line via image tag
+
+`podman pull ghcr.io/ansible-community/community-ee-minimal:latest`
+`podman pull ghcr.io/ansible-community/community-ee-minimal:<ansible-core-version-minimal-ee-version>`
+
+
+
+## Whats inside community-ee-base?
+
+- `base_image`: fedora-minimal
+
+- `ansible-core`: ansible-core 2.16.0 (latest version of ansible-core)
```

Comment:
use `<version of ansible-core>` to be consistent (and for easier search and replace)